acceleration-everest-03-dotnet/UnitTests/Validators/Create/CreateCustomerValidatorTests.cs

```diff
@@ -0,0 +1,383 @@
+﻿using AppServices.Validators.Create;
+using FluentAssertions;
+using System;
+using UnitTests.Fixtures.Customers;
+
+namespace UnitTests.Validators.Create;
+
+public class CreateCustomerValidatorTests
+{
+    [Fact]
+    public void Should_CreateCustomer_Valid_Successfully()
+    {
+        // Arrange
+        var createCustomerTest = CreateCustomerFixture.GenerateCreateCustomerFixture();
+        var validCreateCustomer = new CreateCustomerValidator();
+
+        // Action
+        var result = validCreateCustomer.Validate(createCustomerTest);
+
+        // Assert
+        result.IsValid.Should().BeTrue();
+    }
+
+    [Fact]
+    public void Should_NotCreateCustomer_When_FullName_Empty()
+    {
+        // Arrange
+        var createCustomerTest = CreateCustomerFixture.GenerateCreateCustomerFixture();
+        createCustomerTest.FullName = "";
+        var validCreateCustomer = new CreateCustomerValidator();
+
+        // Action
+        var result = validCreateCustomer.Validate(createCustomerTest);
+
+        // Assert
+        result.IsValid.Should().BeFalse();
+    }
```

Comment:
Acredito que para testes dos validators, como você tem Validations para cada prop, utilizar o TestValidate ao invés do Validate pois assim, podemos testar se o erro está vindo exatamente onde queremos.
```suggestion
    public void Should_NotCreateCustomer_When_FullName_Empty()
    {
        // Arrange
        var createCustomerTest = CreateCustomerFixture.GenerateCreateCustomerFixture();
        createCustomerTest.FullName = "";
        var validCreateCustomer = new CreateCustomerValidator();

        // Action
        var result = validCreateCustomer.TestValidate(createCustomerTest);

        // Assert
        result.ShouldHaveValidationErrorFor(x => x.FullName);
    }
```

Reply:
Alterado em todos os testes de validator.